shard-operator/src/main/kubernetes/sample.yml

```diff
@@ -1,11 +1,11 @@
 apiVersion: com.redhat.service.bridge/v1alpha1
 kind: BridgeIngress
 metadata:
-  name: 99259d8b-3fdd-48de-a810-a66db8b9914c
+  name: my-bridge-ingress
```

Comment:
Since we do create a `deployment` and a `service` with the same `BridgeIngress.metadata.name`, it must be a valid name for such resources. `99259d8b..` is not a valid Service name. 

Revisit this logic https://issues.redhat.com/browse/MGDOBR-116